index.html

```diff
@@ -124,6 +124,89 @@ <h2><dfn>CropTarget Production</dfn></h2>
     </section>
     <section id="restricting-a-track">
       <h2><dfn>Restriction Mechanism</dfn></h2>
+      <section>
+        <h3>Definitions</h3>
+        <section>
+          <h4>Restrictable tracks</h4>
+          <p>
+            We say that a {{MediaStreamTrack}} <var>T</var> is a
+            <dfn>restrictable MediaStreamTrack</dfn> if and only if it fulfils all of the following
+            conditions:
+          </p>
+          <ul>
+            <li>
+              <var>T</var> is either a {{MediaStreamTrack}} returned inside of a {{MediaStream}}
+              through a call to {{MediaDevices/getDisplayMedia()}}, or a clone of such a track.
+            </li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-kind">kind</a> is
+              <a data-cite="mediacapture-streams/#dfn-video">"video"</a>.
+            </li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-readystate">readyState</a> is
+              <a data-cite="mediacapture-streams/#idl-def-MediaStreamTrackState.live">"live"</a>.
+            </li>
+            <li><var>T</var> is not <a data-cite="mediacapture-region#dfn-cropped">cropped</a>.</li>
+          </ul>
+        </section>
+        <section>
+          <h4>Elements eligible for restriction</h4>
+          <p>
+            We say that an {{Element}} <var>E</var> is <dfn>eligible for restriction</dfn> if and
+            only if it fulfils all of the following conditions:
+          </p>
+          <ul>
+            <li><var>E</var> forms a stacking context.</li>
+            <li>
+              <var>E</var> is
+              <a data-cite="css-transforms-2#grouping-property-values">flattened in 3D.</a>
+            </li>
+            <li>
+              <var>E</var> forms a <a data-cite="filter-effects-2/#backdrop-root">backdrop root</a>.
```

Comment:
Also:

- "has only one [=box fragment=]".
- "is <a href="https://www.w3.org/TR/css-images-4/#element-not-rendered">rendered</a>".

Reply:
* Rendered - done.
* One box fragment - I think this is the same issue as was discussed on another thread; please correctly me if I got it wrong. A fragmented element still has a bounding box and the fragments can be all layered together as a captured frame. In practice, user agents might not wish to prioritize implementing this niche case, but in theory, I believe this can be allowed by the spec.

Reply:
Added one box fragment too.